src/App.tsx

```diff
@@ -1,24 +1,143 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  ChangeEvent,
+  useEffect,
+  useState,
+} from 'react';
+
+import { getTodos, addTodo, removeTodo } from './api/todos';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Filter } from './components/Filter/Filter';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import NotificationError from
+  './components/NotificationError/NotificationError';
 
-const USER_ID = 0;
+const USER_ID = 6846;
 
 export const App: React.FC = () => {
+  const [task, setTask] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [tempTodo, setTemptodo] = useState<Todo | null>(null);
+  const [isDataUpdated, setIsdataUpdated] = useState(false);
+  const [activeIds, setActiveIds] = useState([0]);
+
+  const handleTodoChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setTask(event.target.value);
+  };
+
+  const loadTodos = async () => {
+    setIsdataUpdated(true);
+
+    if (task) {
+      setTemptodo({
+        id: 0,
+        userId: USER_ID,
+        title: task,
+        completed: false,
+      });
+    }
+
+    try {
+      await getTodos(Number(USER_ID)).then(res => {
+        setTodos(res);
+        setTemptodo(null);
+        setIsdataUpdated(false);
+      });
+    } catch {
+      setError('unable to get todos');
+    }
+  };
+
+  const handleTodoSubmit = async (event: ChangeEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (task === '') {
+      setError("The title can't be empty");
+
+      return;
+    }
+
+    try {
+      await addTodo({
+        userId: USER_ID,
+        title: task,
+        completed: false,
+      }).then(() => loadTodos());
```

Comment:
Why do you use async/await with then? Choose only one approach to work with asynchronous functions